explorer.go

```diff
@@ -29,30 +29,36 @@ func (c *appContext) explorerUI(w http.ResponseWriter, r *http.Request) {
 
 	idx := c.BlockData.GetHeight()
 
-	start, errE := strconv.Atoi(r.URL.Query().Get("start"))
-	if errE != nil || start == 0 || start > idx {
-		start = idx
+	end, errE := strconv.Atoi(r.URL.Query().Get("end"))
+	if errE != nil || end == 0 || end > idx {
+		end = idx
 	}
-	end, errS := strconv.Atoi(r.URL.Query().Get("end"))
-	if errS != nil || end == 0 || start-end < 10 {
-		end = start - 26
-	} else if start-end > 200 {
-		end = start - 200
+	start, errS := strconv.Atoi(r.URL.Query().Get("start"))
+	if errS != nil || start == 0 || end-start < 10 {
```

Comment:
we should have data for block 0 (genesis block), although I can't recall how this is handled.  If there's nothing breaking about the data in genesis, we should not prevent it from being queried